.travis.yml

```diff
@@ -11,8 +16,18 @@ perl:
 env:
   - ALIEN_INSTALL_TYPE=share
   - ALIEN_INSTALL_TYPE=system
+matrix:
+  allow_failures:
+    - perl: 5.8
+      env: ALIEN_INSTALL_TYPE=share
```

Comment:
The config script that ships with openssl apparently requires 5.10 now.  Should still work for a system install though so I just allowed a failure for 5.8 share install.